best_practices/introduction.rst

```diff
@@ -95,3 +103,4 @@ practices**. The reasons for not doing it are various:
   your tests or adding features that provide real value to the end users.
 
 .. _`Fabien Potencier`: https://connect.sensiolabs.com/profile/fabpot
+.. _`download and install`: http://symfony.com/download
```

Comment:
Because in this case we are talking about downloading and installing the installer. The symfony.com/download page is the place where this information is more concisely explained. The installation chapter is more verbose because it explains everything.